lib/bunch/macro.ex

```diff
@@ -82,4 +82,17 @@ defmodule Bunch.Macro do
 
     ast
   end
+
+  @doc """
+  Takes a code block, expands macros inside and pretty prints it.
+  """
+  defmacro peek_code(do: block) do
+    Macro.prewalk(block, fn tree -> Macro.expand(tree, __ENV__) end)
+    |> Macro.to_string()
+    |> IO.puts()
+
+    quote do
```

Comment:
`quote do unquote(x) end == x` for any x... Moreover, returning this usually results in executing the given code, which I wouldn't expect from this function. Maybe just return what `IO.puts` returns

Reply:
My intention is to make this function as transparent as `IO.inspect/3` so it can be plopped in without meaningful consequences of what code does. 

Code in block `x` is not executed.
https://rextester.com/VAY68432

Reply:
Of course, it's not executed in the compile time, but it is once `Test.test` is evaluated. I thought this macro is meant to be used in iex, where it leads to the immediate evaluation of the code. Now I understand it's desired, but in this case, I would move it to the new module `Bunch.Code`, that would correspond to elixir's `Code`. There it would fit better.

Reply:
done